src/CXFX.ts

```diff
@@ -0,0 +1,114 @@
+import { Selector } from "./Selector";
+import { genericCleanup, shorten, toFixed } from "./util";
+
+export class CXFX {
+    private tag = "pb-cx";
+    cleanup() {
+        genericCleanup(this.tag);
+    }
+    run() {
+        // CXOB
+        const elements = document.querySelectorAll(Selector.CXOBTable + " > tbody > tr > td");
+        for (let i = 0; i < elements.length; i++) {
+            const element = elements[i];
+            const text = element.textContent;
+            const matches = text && text.match(/^NEO Charter Exploration Market Maker$/);
```

Comment:
This I'm guessing isn't portable enough to be merged in for everyone

Reply:
I can try to put switches for things like these in the sidebar but none of those buttons work for me so it will take some time to figure out, if even possible